docs/src/pages/articles.js

```diff
@@ -67,7 +65,41 @@ export default function ArticlesPage() {
   )
 }
 
-const fetcher = async () => {
-  const { data } = await axios.get('/api/medium')
-  return data
+export async function getStaticProps() {
+  try {
+    const { data } = await axios({
+      method: 'GET',
+      url: 'https://medium.com/feed/hyperjump-tech',
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Content-Type': 'application/rss+xml',
+      },
+    })
+
+    if (!data) {
```

Comment:
we can throw exception here to short circuit to the same return statement in catch block.

Reply:
Whoopsie. Didn't think about that. Should be fixed